deploy/console-branding/cr-console.operator.config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: operator.openshift.io/v1
+kind: Console
+metadata:
+  name: cluster
+spec:
+  managementState: Managed
+  authentication:
+    # the logout redirect must be a valid URI ethat starts with http(s)
+    # valid examples: http://localhost:8080/logout.html, https://www.redhat.com/log/out/
+    logoutRedirect: "https://www.redhat.com/i/should/logout"
+  customization:
+    brand: dedicated 
+    # the documentation base must be a valid URI that starts with http(s) AND it must have a trailing slash
+    # the trailing slash is required because additional values will be appended
+    # valid examples: https://localhost:9999/docs/, http://192.168.1.5:3000/doc/location/
+    documentationBaseURL: "https://docs.openshift.com/dedicated"
```

Comment:
@liamwazherealso I was able to get this applied finally on a new 4.0.0-0.5 cluster.  I didn't see the change land until I added a trailing `/` to the `documentationBaseURL` property making the value `https://docs.openshift.com/dedicated/`.  This is consistent with the config that is on the sandbox cluster you have this working on.  Not sure if it was timing or that change that landed this, but since it works I recommend we update the PR.

Reply:
This error was silly on my part. The CR I used while testing differed in this field. Silly oversight not using the exact one in the repo. Updated.